### PR TITLE
Add Trakt Trending List support

### PR DESCRIPTION
**Pull request by mza921**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # Plex Auto Collections
-##### Version 2.1.1
+##### Version 2.2.0
```
Comment (meisnate12) [inline README.md:2]: in plex_auto_collections.py right after the big title it prints out there's another version number

**Review by meisnate12 — COMMENTED** [review]
looks awesome we just need the version in plex_auto_collections.py

File: app/plex_tools.py [hunk 2]
```diff
@@ -118,7 +118,7 @@ def add_to_collection(config_path, plex, method, value, c, map, subfilters=None)
     shows = []
     items = []
     missing = []
-    if (method == "trakt_list" or ("tmdb" in method and plex.library_type == "show")) and not TraktClient.valid:
+    if ("trakt" in method or ("tmdb" in method and plex.library_type == "show")) and not TraktClient.valid:
```
[the remark this answers is not in the document]
Comment (meisnate12) [reply app/plex_tools.py:121]: nevermind i see why you did it

File: README.md [hunk 3]
```diff
@@ -156,8 +157,10 @@ The only required attribute for each collection is the list type. There are eigh
 - [TVDb Show](#tvdb-show-list-type)
 - [IMDb List or Search](#imdb-list-or-search-list-type)
 - [Trakt List](#trakt-list-list-type)
+- [Trakt Trending List](#trakt-trending-list-list-type)
+- [Tautulli List (List Type)](#tautulli-list-list-type)
 
-Note that each list type supports multiple lists.
+Note that each list type supports multiple lists, with the `Trakt Trending List` as the only exception.
```
[the remark this answers is not in the document]
Comment (mza921) [reply README.md:163]: Thanks, I'll update

File: README.md (hunk 1 above)
Comment (mza921) [reply README.md:2]: Will also update this.

**Pull request merged**